webpack/assets/javascripts/react_app/components/HostDetails/Audits/index.js

```diff
@@ -1,143 +1,101 @@
-                                            }}
-                                          >
-                                            {value[0]}
-                                          </mark>
-                                        </ElipsisWithTooltip>
-                                      )}
-                                    </DataListCell>,
-                                    <DataListCell
-                                      style={{
-                                        width: '350px',
-                                        overflowX: 'hidden',
-                                      }}
-                                      key={`new-value-${i}`}
-                                    >
-                                      {value && (
-                                        <mark
-                                          style={{
-                                            backgroundColor: '#90EE90',
-                                          }}
-                                        >
-                                          {value[1]}
-                                        </mark>
-                                      )}
-                                    </DataListCell>,
-                                  ]}
-                                />
-                              </DataListItemRow>
-                            </DataListItem>
-                          )
-                        )}
-                    </DataList>
-                  </AccordionContent>
-                </AccordionItem>
-              );
-            return null;
-          })}
-        </Accordion>
+        <SkeletonLoader
+          skeletonProps={{ count: 3 }}
+          status={status}
+          emptyState={
+            <Bullseye>
+              <Title headingLevel="h4"> {__('No Results found')} </Title>
+            </Bullseye>
+          }
+        >
+          <DataList aria-label="recent-audit-table" isCompact>
+            {audits?.map(
+              ({
+                user_name: user,
+                user_id: userID,
+                created_at: timestamp,
+                action,
+                remote_address: ip,
+                id,
+              }) => (
+                <DataListItem aria-labelledby="simple-item1">
+                  <DataListItemRow>
+                    <DataListItemCells
+                      dataListCells={[
+                        <DataListCell key={`id-${action}`}>
+                          {action}
+                        </DataListCell>,
+                        <DataListCell key={`date-${id}`}>
+                          <RelativeDateTime date={timestamp} />
+                        </DataListCell>,
+                        <DataListCell
+                          wrapModifier={DataListWrapModifier.truncate}
+                          key={`user-${id}`}
+                        >
+                          <a href={foremanUrl(`/users/${userID}/edit`)}>
```

Comment:
users without `edit_users` permission won't be able to follow this link, and in any case I think we don't need it here - if needed and they have the right permission they'll be able to get to the user from the all audits view